src/Maestro/Maestro.Web/Helpers.cs

```diff
@@ -99,5 +101,60 @@ public static async Task<IActionResult> ProxyRequestAsync(this HttpContext conte
             }
 
         }
+
+        public static Dictionary<string, KustoQuery> CreateBuildTimesQueries(string repository, string branch, int days)
+        {
+            var parameters = new List<KustoParameter> {
+                new KustoParameter("_Repository", KustoDataTypes.String,  repository.Split("/").Last()),
+                new KustoParameter("_SourceBranch", KustoDataTypes.String, branch),
+                new KustoParameter("_Days", KustoDataTypes.TimeSpan, $"{days}d")
+            };
+
+            string publicProject = "public";
+
+            // Builds in AzDo are only found in the internal project
+            if (repository.Contains("dev.azure.com"))
```

Comment:
Is this correct? For default channels that specify github as their repo, the builds still occur in AzDO. Would those be reported as being on the public project?

Reply:
In Kusto, yes. There are no "public" builds for AzDo repos, all happen in internal. However, they do have PRs (in internal). This if is catching that case. For github repos, their default channel uses the github url, their PR runs happen in public, and their official builds happen in internal.